python/pyarrow/libarrow_substrait.pxd

```diff
@@ -0,0 +1,26 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Small comment: Judging by similar files in Arrow, this file should probably be located under `python/pyarrow/includes`.